src/sentry/utils/pytest/relay.py

```diff
@@ -94,7 +96,7 @@ def relay_server_setup(live_server, tmpdir_factory):
     container_name = _relay_server_container_name()
     _remove_container_if_exists(docker_client, container_name)
     options = {
-        "image": "us.gcr.io/sentryio/relay:nightly",
+        "image": f"us.gcr.io/sentryio/relay:{RELAY_DOCKER_TAG}",
```

Comment:
I would maybe rename this to `RELAY_TEST_IMAGE` and point to the image, instead of just the tag.

Reply:
Works for me. I like it.